.github/workflows/master.yml

```diff
@@ -0,0 +1,36 @@
+# Push to master workflow.
+#
+# Less strict than the PR workflow because checks have run already.
+
+name: Master
+
+on:
```

Comment:
Can we can change this to:
```
on:
 push:
  branches:
   - master
```
Since we don't push directly to the master branch.

(Unless a PR merge doesn't trigger the push event)

Reply:
I'm not sure I know what you mean @lionel1704. And yeah, I think PR merges don't trigger a push event.

Reply:
I just checked on my repo. PR merges are triggering the push event on the master branch.
So we don't actually need the conditional for merged pull requests. Instead we can trigger this workflow using the `push` event. Does that make sense?

Reply:
Ah that's great, will make the change! I guess a lot of the info I've been reading is out of date.